test/DynamoCoreWpfTests/WorkspaceOpeningTests.cs

```diff
@@ -95,11 +95,12 @@ public void OpeningWorkspaceDoNotStartWithUnSavedChanges()
             Assert.AreEqual(ws.HasUnsavedChanges, false);
         }
 
-        [Test]
+        [Test, Ignore]
```

Comment:
After discovering `hasRunWithoutCrash` is defaulting to `true` and discussing issues found in [QNTM-2839](https://jira.autodesk.com/browse/QNTM-2839) with @QilongTang we have to decided to Ignore this test until we are able to make the necessary changes to how our serialization tests operate.  I have noted the issue in the code and added a note to QNTM-2839 about restoring this test when the required modifications are made.